meta-lmp-base/conf/distro/lmp.conf

```diff
@@ -5,6 +5,9 @@ DISTRO_NAME = "Linux-microPlatform"
 
 IMAGE_LINGUAS ?= "en-us"
 
+# Include distro features in pre-build configuration output
+BUILDCFG_VARS += "DISTRO_FEATURES"
```

Comment:
Why not in `meta-lmp-base/conf/distro/include/lmp.inc` and would be available in all distros

Reply:
Yup, had that in my mind when changing but ended up changing at the wrong place :-)

Reply:
Updated.